houston/houston.go

```diff
@@ -36,6 +34,18 @@ type Request struct {
 	Variables map[string]interface{} `json:"variables"`
 }
 
+//Do (request) is a wrapper to more easily pass variables to a client.Do request
+func (r *Request) DoV2(api *Client) (*Response, error) {
+	doOpts := httputil.DoOptions{
+		Data: r,
+		Headers: map[string]string{
+			"Accept": "application/json",
+		},
+	}
+
+	return api.Do(doOpts)
+}
+
 // Do (request) is a wrapper to more easily pass variables to a client.Do request
 func (r *Request) Do() (*Response, error) {
```

Comment:
Do we need a "V2" here - (i.e. do we still use `Do()` anywhere?) If we want to keep both then `DoWithClient` is probably a better name than `DoV2`


And `Do` could be implemented as `return DoWithClient(NewHoustonClient(httputil.NewHTTPClient()))`

Reply:
done.